src/measure_theory/order/upper_lower.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Kexing Ying
+-/
+import measure_theory.covering.besicovitch_vector_space
+import topology.algebra.order.upper_lower
+
+/-!
+# Order-connected sets are null-measurable
+
+This file proves that order-connected sets in `ℝⁿ` under the pointwise order are measurable.
```

Comment:
Can you recall what order-connected means here? The concept is clear to me in linear orders, less clear in general orders. In particular, a few examples would help: the PR shows that the set is null-measurable because the boundary has measure zero, but is there an example showing that it could be non-measurable?